### PR TITLE
[10] Pundit introduction

### PR DESCRIPTION
**Pull request by justynapaldyna**
Hello! I am working on using Pundit, it does work already (i checked as a regular user and it forbiddens actions) but still some things to manage, views, redirecting back if not admin, etc. :))

### CODE REVIEW

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -17,11 +17,13 @@ def new
 
   # GET /categories/1/edit
   def edit
+    
   end
 
   # POST /categories or /categories.json
   def create
     @category = Category.new(category_params)
+    authorize @category
```
Comment (paleksandrowicz) [inline app/controllers/categories_controller.rb:26]: You should always authorize first

**Comment by justynapaldyna** [comment]
Hi, i tried to authorize users only to edit and delete own comments unless theyre an admin. I came with this in CommentsPolicy: 
def destroy?
user.id == record.user_id || user.admin?
end
BUT it keeps telling me that there is no method 'user_id' for comment:class :< I dont have an idea what is wrong, because i have 'user_id" field in Comment schema

File: app/policies/comment_policy.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    def update?
+      record.user_id == user.id
+    end
+  
+    def edit?
+      update?
+    end
+  
+    def destroy?
+      record.user_id == user.id || user.admin?
```
Comment (paleksandrowicz) [inline app/policies/comment_policy.rb]: are you sure you are passing a record to this policy?

Comment (paleksandrowicz) [reply app/policies/comment_policy.rb]: ^ this is regarding your pr question

**Review by paleksandrowicz — APPROVED** [review]
Looks good, sorry that took me so long, but I had to move out :D

**Pull request merged**